codelists/api.py

```diff
@@ -166,11 +168,17 @@ def versions(request, codelist):
 
     try:
         if "codes" in data:
-            clv = create_version_with_codes(
-                codelist=codelist,
-                codes=set(data["codes"]),
-                tag=data.get("tag"),
-            )
+            if "force_create" in data:
```

Comment:
I don't think this needs to be checked, we can just always pass the `force_create` parameter, with a defaul:
```
clv = create_version_with_codes(
                codelist=codelist,
                codes=set(data["codes"]),
                tag=data.get("tag"),
                force_create=data.get("force_create", False)
            )
```